catalog-info.yaml

```diff
@@ -1,30 +1,25 @@
 # Backstage documentation
 # https://backstage.io/docs/features/software-catalog/descriptor-format/
 
-# !!! WARNING !!!
-# This is a template file with a number of fields that need to be filled before merging this to the default branch
 apiVersion: backstage.io/v1alpha1
 # Component, API, Template, Group, User, Resource, System, Domain, Location
-kind: unknown
+kind: API
 metadata:
   name: boilerplate-php 
-  description: unknown
+  description: An example app to get started with using the Contentful PHP SDK.
   annotations:
     github.com/project-slug: contentful/boilerplate-php 
-    contentful.com/service-tier: "unknown" #1, 2, 3, 4
+    contentful.com/service-tier: "1" #1, 2, 3, 4
 
   tags:
-    - update-me
-    #need to add sast.yaml to .github/workflows and enable it in polaris dashboard
-    #once that is done this can be changed to sast-enabled
     - sast-disabled
     #make this match the value from service-tier above
-    - tier-unknown
+    - tier-1
 spec:
   #cli, component, contentful.com/template, documentation, function, library, service, template, website
-  type: unknown 
+  type: contentful.com/template 
   #deprecated, experimental, production, unknown
-  lifecycle: unknown  
-  system: unknown #optional
+  lifecycle: production  
```

Comment:
i suspect `unknown` is sufficient here for an example/boilerplate template repo

Reply:
the backstage docs describe spec as 
```
The software catalog accepts any lifecycle value, but an organization should take great care to establish a proper taxonomy for these.

The current set of well-known and common values for this field is:

experimental - an experiment or early, non-production component, signaling that users may not prefer to consume it over other more established components, or that there are low or no reliability guarantees
production - an established, owned, maintained component
deprecated - a component that is at the end of its lifecycle, and may disappear at a later point in time
```

I don't think we have an internally defined taxonomy for this but I think an argument could be made these SDKs are production components.

Reply:
okay then this is acceptable given that criteria, thank you

Reply:
I'd argue the same - this is maintained, owned and can be used, therefore production seems fitting.